bip-newproposal.mediawiki

```diff
@@ -0,0 +1,607 @@
+   ECDSA Signature: 0x304402201234567890abcdef1234567890abcdef1234567890abcdef1234567890abcdef022012345678901234567890abcdef1234567890abcdef1234567890abcdef12345678
+
+   # Complete QES2 Signature
+   QES2 Signature: [Dilithium Signature || ECDSA Signature]
+
+Security Considerations
+======================
+
+Quantum Timeline
+---------------
+
+The implementation timeline considers quantum computing development:
+
+1. **Near-term (3-5 years)**: Low quantum threat, preparation phase
+2. **Medium-term (5-10 years)**: Growing threat, transition phase
+3. **Long-term (10+ years)**: Potential quantum capacity to break ECDSA
+
+The risk assessment formula can be modeled as:
+
+.. code-block::
+
+   Risk = Probability of Attack × Impact of Compromise
+
+Where the probability increases with quantum computing development, and the impact depends on the value secured by ECDSA.
+
+Migration Security
+----------------
+
+During the transition period:
+
+1. Coins in traditional addresses will become increasingly vulnerable
+2. Tools should be provided to securely migrate funds to quantum-resistant addresses
+3. Community education about quantum security risks becomes critical
+
+The security of funds during migration depends on the following inequality:
+
+.. code-block::
+
+   T_migration < T_quantum break
+
+Where T_migration is the time required to migrate funds to quantum-resistant addresses, and T_quantum break is the time until quantum computers can break ECDSA.
+
+Implementation Security
+----------------------
+
+Implementers should be aware of:
+
+1. Side-channel resistance in both signature schemes
+2. Appropriate entropy for key generation
+3. Secure storage of larger keys and signatures
+4. Protection against combined cryptographic attacks
+
+The entropy requirements for key generation are:
+
+.. code-block::
+
+   Entropy_QES2 ≥ max(Entropy_ECDSA, Entropy_Dilithium) ≥ 256 bits
+
+Security Proofs
+--------------
+
+The security of QES2 relies on the following propositions:
+
+1. **Dilithium Security**: The Dilithium signature is secure against quantum adversaries under the hardness assumptions of Module-LWE and Module-SIS problems.
+
+2. **ECDSA Security**: While vulnerable to quantum attacks, ECDSA remains secure against classical adversaries.
+
+3. **Binding Property**: The ECDSA signature validates the Dilithium signature, creating a binding that requires breaking both schemes or finding hash collisions to forge.
+
+4. **Hybrid Security Theorem**: If either signature scheme remains secure, the combined QES2 signature remains secure. Formally:
+
+   .. code-block::
+
+      Adv_QES2(A) ≤ min(Adv_ECDSA(A'), Adv_Dilithium(A''))
+
+   Where Adv_X(A) represents the advantage of an adversary A against scheme X.
+
+Acknowledgments
+==============
+
+This proposal builds on the work of several other BIPs, including BIP-340, BIP-341, and BIP-342 (Taproot), and incorporates concepts from ongoing research in post-quantum cryptography for blockchains.
```

Comment:
I’m confused that this proposal mentions BIP 340 several times, but insufficiently addresses BIP 340 signatures.